app/lib/actions/katello/product/content_destroy.rb

```diff
@@ -9,10 +9,19 @@ def plan(repository)
                         owner: repository.product.organization.label,
                         product_id: repository.product.cp_id,
                         content_id: repository.content_id)
+
+            katello_content_id = repository.product.product_content_by_id(repository.content_id).content_id
+            plan_action(Katello::ProductContent::Remove,
+                        product_id: repository.product.id,
+                        katello_content_id: katello_content_id)
+
             if repository.other_repos_with_same_content.empty?
               plan_action(Candlepin::Product::ContentDestroy,
                           owner: repository.product.organization.label,
                           content_id: repository.content_id)
+
+              plan_action(Katello::Content::Destroy,
+                          katello_content_id: katello_content_id)
```

Comment:
I don't think these destroys need to be in their own action, (and shouldn't be done as part of the run phase regardless).  Any reason to not just delete these objects here?